### ISSUE TITLE
Why blockquote for Prerequisites?

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/DEPRECATED-lesson-template#25]
@gvwilson Why

```
> ## Prerequisites
>
> A short paragraph describing what learners need to know
> before tackling this lesson.
```

instead of

```
## Prerequisites

A short paragraph describing what learners need to know
before tackling this lesson.
```

If you want some style for it we should use

```
> ## Prerequisites {.prerequisites}
>
> A short paragraph describing what learners need to know
> before tackling this lesson.
```

**Comment by gvwilson** [comment]
1. A blockquote because we want to apply a style to the whole thing and Markdown doesn't have `div`.
2. My thought was that we could inject the `{.prerequisites}` attribute during processing (because the title `Prerequisites` has to be there), and save people having to provide duplicate information - same with learning objectives.

**Comment by aaren** [comment]
How do we close this? 

There is an inconsistency between the `README`, which states:

```
`index.md` must be structured as follows:

    ---
    layout: lesson
    title: Lesson Title
    ---
    Paragraph(s) of introductory material.

    > ## Prerequisites {.prereq}
    >
    > What learners need to know before tackling this lesson.
```

and `index.md`, which contains:

```
---
layout: page
title: Lesson Title
---
Paragraph of introductory material.

> ## Prerequisites
>
> A short paragraph describing what learners need to know
> before tackling this lesson.
```

I can add `.prereqs` to the special classes list so that it gets converted to a div, allowing css to be applied to it, and update `index.md`. This would be consistent with how we are doing it for the other blockquote divs and would close this issue.

**Comment by rgaiacs** [comment]
@aaren The inconsistency is because

> My thought was that we could inject the `{.prerequisites}` attribute during processing (because the title `Prerequisites` has to be there), and save people having to provide duplicate information - same with learning objectives.

To close this we need to change `tools/ blockquote2div.py` to it add the class `prerequisites` to every `blockquote`/`div` starting with the title `Prerequisites`.

**Comment by rgaiacs** [comment]
Closed by #37.
